static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.js

```diff
@@ -45,7 +45,7 @@ const fillAdvertSlots = (
     start: () => void,
     stop: () => void
 ): Promise<void> => {
-    if (commercialFeatures.dfpAdvertising) {
+    if (commercialFeatures.dfpAdvertising || commercialFeatures.adFree) {
```

Comment:
Yes. Merchandising expressed an interest in targeting specific inventory to ad-free users.